pkg/controllers/et/createet_reconciler.go

```diff
@@ -0,0 +1,107 @@
+/*
+Copyright [2021-2023] Micro Focus or one of its affiliates.
```

Comment:
We recently updated the copyright text. 
```suggestion
Copyright [2021-2023] Open Text.
```